language/tools/test_generation/src/summaries.rs

```diff
@@ -13,10 +14,10 @@ use crate::{
 use vm::file_format::{Bytecode, Kind, SignatureToken};
 
 /// A `Precondition` is a boolean predicate on an `AbstractState`.
-type Precondition = dyn Fn(&AbstractState) -> bool;
+type Precondition = dyn Fn(&AbstractState) -> Result<bool, VMError>;
```

Comment:
As discussed, this should probably stay bool.

Reply:
Addressed.